src/textual/messages.py

```diff
@@ -41,3 +41,12 @@ class CursorMove(Message):
     def __init__(self, sender: MessagePump, line: int) -> None:
         self.line = line
         super().__init__(sender)
+
+
+@rich.repr.auto
+class RefreshStyles(Message):
```

Comment:
One thing I've seen done in message passing systems is to name messages after the "thing that happened" (e.g. `StyleClassesUpdated`), rather than having the sender say what they *want* to happen (`RefreshStyles`). This means a consumer isn't restricted to doing just what they're asked by the sender - they can handle an event however they wish.

At a conceptual level, when you send a message or event, are you saying "here's what happened", or are you saying "whoever is listening, please perform some operation"? I think it's nice to agree on these conventions early.